packages/apollo/src/generate.ts

```diff
@@ -72,7 +72,19 @@ export default function generate(
     const outputIndividualFiles =
       fs.existsSync(outputPath) && fs.statSync(outputPath).isDirectory();
 
-    const generator = generateSwiftSource(context, outputIndividualFiles, only);
+    var suppressSwiftMultilineStringLiterals: boolean;
+    if (options.suppressSwiftMultilineStringLiterals == true) {
+      suppressSwiftMultilineStringLiterals = true;
+    } else {
+      suppressSwiftMultilineStringLiterals = false;
+    }
```

Comment:
Typescript frenz: I was doing this late at night - is there a better way to "if false or undefined use this default value" than this? My googling skills on Typescript aren't great.

Reply:
Since the default value is indeed `false` you can just say `suppressSwiftMultilineStringLiterals = options.suppressSwiftMultilineStringLiterals || false;`. This will work for any optional type where the default value is the same as the falsy value of the type (if there is any at all), e.g. it would work for `string?` if the default value is `""`.

I think you can also say `!!options.suppressSwiftMultilineStringLiterals` but I'm not sure offhand if TypeScript likes that idiom.

Reply:
The default value is `false` when passed in from the command line but there's a couple places where this struct is created without any properties - that's why this has to be `suppressSwiftMultilineStringLiterals?: boolean` rather than `suppressSwiftMultilineStringLiterals: boolean`

Yes I agree this is silly

Reply:
@designatednerd I believe you could use the `Boolean` constructor here too. Which would play a little nicer than the `!!` with ts and would prevent you from having to define the type explicitly.

```
    const suppressSwiftMultilineStringLiterals = Boolean(options.suppressSwiftMultilineStringLiterals)
```

this would probably also work if you need to check if its explicitly `true` and not just something truthy like a string.

```
const suppressSwiftMultilineStringLiterals = options.suppressSwiftMultilineStringLiterals === true;
```

unless I'm missing something here.

Reply:
Also, any reason you're using `var` instead of `const` or `let` here?

Reply:
I think because I thought I had to because I wasn't assigning immediately

Reply:
FWIW the [MDN docs on the `Boolean` object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/boolean) explicitly describes `!!` as equivalent to using `Boolean` as a function.